release/build-common.xml

```diff
@@ -195,13 +198,14 @@
                     <arg value="Cocoa"/>
                     <arg line="${additional.jni.frameworks}"/>
 
-                    <arg value="-F/${SDKroot}/System/Library/Frameworks/JavaVM.framework/Versions/A/Frameworks"/>
-                    <arg value="-framework"/>
-                    <arg value="JavaRuntimeSupport"/>
                     <arg value="-F${JNFdir}"/>
                     <arg value="-framework"/>
                     <arg value="JavaNativeFoundation"/>
 
+                    <arg value="-F${SDKroot}/System/Library/Frameworks"/>
+                    <arg value="-framework"/>
+                    <arg value="JavaRuntimeSupport"/>
+
```

Comment:
reorder this, because JNF needs to take priority to build, and path to the framework in the SDK